examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -28,7 +28,7 @@
    "id": "854815b0",
```

Comment:
<p>not sure if it makes any difference but, shall we capitalize <code>merlin</code> to <code>Merlin</code> like below?  </p><p> </p><p>img src="https://developer.download.nvidia.com/notebooks/dlsw-notebooks/Merlin_models_ecommerce-session-based-next-item-prediction-for-fashion/nvidia_logo.png" style="width: 90px; float: right;"&gt; </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I asked and he said capitalization makes no differences in tracking</p>